### DISCUSSION TITLE
Nickname overflows in the "New Chat" drawer if it's long

### DISCUSSION THREAD

**Discussion by smohamedjavid** [status-im/status-legacy#16687] [General]
# Bug Report

The nickname of the contact overflows in the "New Chat" drawer if the nickname is long.

<img src="https://github.com/status-im/status-mobile/assets/19339952/ef54755b-29d2-4f89-8c93-08b5282e59a7" width="300" />

### Expected behaviour

The nickname should not overflow.

#### Prerequisites: Have at least one mutual contact

1) Open Status
2) Set/Update the nickname of the contact to the longest one
3) Navigate to the `Messages` tab
4) Tap on `+` button on top right corner
5) Tap on `New Chat`
6) Observe the name of the contact

### Additional Information

- Status Version: nightly [12/06]
- Operating System: Android, iOS

#### NOTES:

Please check with the Design Team (@status-im/design-team) on the behaviour before starting to work on this issue.

**Comment by Francesca-G** [comment]
@smohamedjavid thanks for sharing this!

Here's the [Figma frame](https://www.figma.com/file/7KIYbhoqNGAIFonE0w9TDz/Messages-for-Mobile?type=design&node-id=8836%3A546523&mode=design&t=8xhENciTcNBGBDtV-1) with the design solution to long nicknames and / or names. 
As you can see it should end with ellipses to avoid any overflow.

Hope this helps :)

<img width="503" alt="Screenshot 2023-07-17 alle 11 02 09" src="https://github.com/status-im/status-mobile/assets/122543089/c7d071dc-e566-4c75-ad30-3a5b0c1ff44d">
